### PR TITLE
update README

### PR DESCRIPTION
**Pull request by cnek**

### CODE REVIEW

**Review by uxd-vincent — DISMISSED** [review]

**Comment by acamill** [comment]
Screenshot 2023-08-23 at 6 51 04 AM
Not sure about this one

**Review by acamill — APPROVED** [review]
Thanks!

File: README.md [hunk 1]
```diff
@@ -2,18 +2,16 @@
...
 The actual deployed state of each contract should live in a protected master branch. The latest master should always reflect the code deployed to all relevant chains
 
 It currently sits at:
 
-<!-- ### Solana -->
-
 - mainnet-beta `UXD8m9cvwk4RcSxnX2HZ9VudQCEeDH6fRnB4CAP57Dr`
 
-- devnet `55NneSZjuFv6cVDQxYKZ1UF99JoximnzP9aY65fJ4JT9` (Used by CI, this address should be update accordingly in ci files)
+- devnet `BLaLouDQKDDf8T9WCgQpzxCBRtcJ4nbqv7fVQeE9PuCY` (Used by CI, this address should be update accordingly in ci files)
```
Comment (uxd-vincent) [inline README.md:14]: the new address for the devnet public complete deployment is: `CW5VzSk7WC4NPyuNt19VFev9FUHhyk5xxHTj2DUWBexu`
Did you want to keep a separate address just for the CI ?

Comment (acamill) [reply README.md:14]: I'm not sure, I think the CI prob has some sols so as long as we use the same keypair. If it works with the new program no problem for me, @cnek @GregoryNEUT do you guys see any issues?

**Review by uxd-vincent — APPROVED** [review]

**Pull request merged**